Assets/Scripts/UI/AmmoCounter.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) TigardHighGDC
+// SPDX-License SPDX-License-Identifier: Apache-2.0
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class AmmoCounter : MonoBehaviour
+{
+    public TextMeshProUGUI bulletText;

```

Comment:
I believe that this variable needs to be renamed.

Reply:
To what?

Reply:
Just needs to be capitalized right?